vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/FieldSteps.java

```diff
@@ -154,10 +154,13 @@ private void enterTextInField(WebElement element, String text, boolean retry, Ru
         {
             element.clear();
             LOGGER.info("Entering text \"{}\" in element", text);
-            if (webDriverManager.isBrowserAnyOf(Browser.SAFARI) && fieldActions.isElementContenteditable(element))
+            if (fieldActions.isElementContenteditable(element))
```

Comment:
I think this logic should be moved to one class: either `FieldSteps.java` or `FieldActions`